starter-core/src/main/java/io/micronaut/starter/feature/ci/workflows/aws/templates/buildSpec.rocker.raw

```diff
@@ -0,0 +1,42 @@
+@import io.micronaut.starter.options.JdkVersion
+@import io.micronaut.starter.options.BuildTool
+
+@args (
+String projectName,
+JdkVersion jdkVersion,
+BuildTool buildTool
+)
+
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: corretto@jdkVersion.majorVersion()
+  pre_build:
+    commands:
+      - echo Nothing to do in the pre_build phase...
+  build:
+    commands:
+      - echo Build started on `date`
+@if (buildTool.equals(BuildTool.MAVEN)) {
+      - mvn verify
```

Comment:
Use Maven wrapper

Reply:
Why? Maven appears to be pre-installed in CodeBuild. Using the wrapper will make the build to download Maven on every run